README.md

```diff
@@ -129,7 +129,7 @@ Editors:
 - [Visual Studio Code (VSCode)](https://www.drupal.org/node/1419996)
 
 
-## Automated Testing (PHPUnit + PHPCS)
+## Automated Testing (PHPUnit + PHPCS + PHPStan)
```

Comment:
Unrelated change - looks like your merged  in the 8.x-3.x branch wrong?

Reply:
I have now pushed a nice clean branch. Only the four files changed. I have left it as two commits, the first is the same a patch 16 which you reviewed. So you can see the changes I made following review.